native/src/widget/progressbar.rs

```diff
@@ -0,0 +1,166 @@
+//! Display a progressbar
+//!
+//!
+//! [`Progressbar`]: struct.Progressbar.html
+use crate::{
+    layout, Background, Color, Element, Hasher, Layout, Length, Point,
+    Rectangle, Size, Widget,
+};
+
+use std::{hash::Hash, ops::RangeInclusive};
+
+/// A progressbar
+///
+/// # Example
+///
+/// ```
+/// # use iced_native::Progressbar;
+/// 
+/// let value = 50.0;
+/// Progressbar::new(0.0..=100.0, value);
+/// ```
+///
+/// ![Default Progressbar](https://user-images.githubusercontent.com/18618951/71662391-a316c200-2d51-11ea-9cef-52758cab85e3.png)
+#[allow(missing_debug_implementations)]
+pub struct Progressbar {
```

Comment:
I think we should name it `ProgressBar` and also change the example name to `progress_bar`.

Reply:
Sure, will change.

Reply:
Should I rename the module ``progress_bar`` too?

Reply:
Yes!